File: src/main/java/de/maggiwuerze/xdccloader/events/server/ServerCreationEvent.java
package de.maggiwuerze.xdccloader.events.server;

import de.maggiwuerze.xdccloader.events.EntityCreationEvent;
import de.maggiwuerze.xdccloader.model.entity.Server;

public class ServerCreationEvent extends EntityCreationEvent<Server> {

	public ServerCreationEvent(Object source, Server server) {
		super(source, server);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/server/ServerCreationEvent.kt
package de.maggiwuerze.xdccwebloader.events.server

import de.maggiwuerze.xdccwebloader.events.EntityCreationEvent
import de.maggiwuerze.xdccwebloader.model.entity.Server

class ServerCreationEvent(source: Any, server: Server) : EntityCreationEvent<Server?>(source, server)